scripts/sudoku.js

```diff
@@ -1,2 +1,58 @@
 var sudokuGrid = new SudokuGrid("SudokuGrid");
 sudokuGrid.render();
+
+//main number buttons//
+var numberButton = document.createElement('button');
+numberButton.id = 'button1';
+numberButton.innerHTML = "1";
+numberButton.className = 'button';
+document.body.appendChild(numberButton);
+
+var numberButton = document.createElement('button');
+numberButton.id = 'button2';
+numberButton.innerHTML = "2";
+numberButton.className = 'button';
+document.body.appendChild(numberButton);
```

Comment:
Some points here:
1. In programming we always try to be DRY (**D**on't **R**epeat **Y**ourself). There is a lot of repetition here. The only difference between each of these buttons is the number. You could use a loop and reduce the repetition here.
1. you are recreating a variable with the same name `numberButton` every time. This could cause problems and confusion.
1. You are appending the button to the body (`document.body.appendChild`). You should be appending it to the div you have created for this purpose.
1. Remember we are using bootstrap here and they have there own classes for buttons which might be useful here. Maybe something like `btn btn-primary` would look nice. You can add your own class over the top as well to customize it but I'd start with bootstrap